src/Device/Driver.cpp

```diff
@@ -92,6 +92,12 @@ AbstractDevice::PutStandbyFrequency(RadioFrequency frequency,
   return true;
 }
 
+bool
+AbstractDevice::ExchangeRadioFrequencies(OperationEnvironment &env)
+{
+  return true;
```

Comment:
Why return success here? Your code did not actually exchange any frequencies, did it?

Reply:
@MaxKellermann I'm not saying you're wrong, but returning true appears to be the norm in this file. 
The argument you're making could also be made for `PutStandbyFrequency` above, and many others.

Reply:
This method is different, because the caller needs to know the difference between "has succeeded", "has failed" and (new third option) "not implemented", or else the caller cannot decide whether to use the fallback code.